install/README.md

```diff
@@ -15,6 +15,7 @@ The [updateVersion.sh](updateVersion.sh) script is used to update image versions
 * `-c <hub>,<tag>` new ca image
 * `-i <url>` new `istioctl` download URL
 * `-g` create a `git commit` titled "Updating istio version" for the changes
+* `-n` <namespace> namespace in which to install Istio control plane components (defaults to "istio-system")
```

Comment:
Shouldn't the default be the value that was last set in istio.VERSION (i.e., the current value), same as is done with other fields?

Reply:
ack.